grammar/liquid-html.ohm

```diff
@@ -166,13 +167,14 @@ Liquid <: Helpers {
     | liquidRawTagImpl<"raw">
     | liquidRawTagImpl<"javascript">
     | liquidRawTagImpl<"schema">
+    | liquidRawTagImpl<"stylesheet">
     | liquidRawTagImpl<"style">
   liquidRawTagImpl<name> =
-    "{%" "-"? space* name space* tagMarkup "-"? "%}"
+    "{%" "-"? space* (name &endOfIdentifier) space* tagMarkup "-"? "%}"
```

Comment:
The `(name &endOfIdentifier)` part here exists to prevent the collision of `{% style` and `{% styles`, since `space*` could collapse to empty string and tagMarkup accepts strings. 

Rule explained:
- match `name` followed by lookahead of `space | "-%}" | "%}"`.